src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -231,31 +249,29 @@ class LicenceService(
   }
 
   private fun notifyApproval(licenceId: Long, licenceEntity: EntityLicence) {
-    notifyService.sendLicenceApprovedEmail(
-      licenceEntity.comEmail.orEmpty(),
-      mapOf(
-        Pair("fullName", "${licenceEntity.forename} ${licenceEntity.surname}"),
-        Pair("prisonName", licenceEntity.prisonDescription.orEmpty()),
-      ),
-      licenceId.toString(),
-    )
+    licenceEntity.mailingList.forEach {
+      notifyService.sendLicenceApprovedEmail(
+        it.email.orEmpty(),
+        mapOf(
+          Pair("fullName", "${licenceEntity.forename} ${licenceEntity.surname}"),
+          Pair("prisonName", licenceEntity.prisonDescription.orEmpty()),
+        ),
+        licenceId.toString(),
+      )
+    }
   }
```

Comment:
I like this concept - a list of interested observers who get notified.  In future, we might need to a way for COMs to remove themselves from the list.

Reply:
Yes, could have an endpoint for unsubscribing